src/tools/crossgen2/jitinterface/jitwrapper.cpp

```diff
@@ -27,11 +27,11 @@ class CORJIT_FLAGS
     unsigned __int64 corJitFlags;
 };
 
-static const GUID JITEEVersionIdentifier = { /* d609bed1-7831-49fc-bd49-b6f054dd4d46 */
-    0xe2ae5b32,
-    0xa9ab,
-    0x426e,
-    {0xbc, 0x2a, 0xae, 0x1a, 0x88, 0x3e, 0x03, 0x67}
+static const GUID JITEEVersionIdentifier = { /* 1ce51eeb-dfd0-4450-ba2c-ea0d2d863df5 */
```

Comment:
@MichalStrehovsky Am I right by setting both `JITEEVersionIdentifier`s in src/inc/corinfo.h and src/tools/crossgen2/jitinterface/jitwrapper.cpp to the same GUID . They were out of sync previously - the version at jitwrapper.cpp corresponds to the GUID before #26395

Reply:
They're not out of sync - only the comment on top of the GUID is out of sync. @trylek missed updating that part in dotnet/corert#7766 (and I missed it in the code review). Updating the comment and the GUID itself is the right thing to do.

Maybe we should just get rid of those comments. Redundant comments that can get out of sync with what the code does are a time sink...